lucida/botframework-interface/check_credentials.js

```diff
@@ -0,0 +1,123 @@
+var system = require('system')
+var username = system.env["BFW_UID"]
+var password = system.env["BFW_PWD"]
+var on_step = 0
+var loading = false
+var load_timeout = 0
+
+if ( username == undefined || password == undefined ) {
+  console.log("[ERROR] This script should be run from start_interface.sh and not as a stand alone script!!!")
+  phantom.exit(400)
+}
+
+var page = require("webpage").create()
+page.settings.userAgent = 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36'
+page.settings.javascriptEnabled = true
+page.settings.loadImages = false
+phantom.cookiesEnabled = true
+phantom.javascriptEnabled = true
+
+phantom.onError = function(msg, trace) {
+  var msgStack = ['[PHANTOM ERROR] ' + msg]
+  if (trace && trace.length) {
+    msgStack.push('TRACE:')
+    trace.forEach(function(t) {
+      msgStack.push(' -> ' + (t.file || t.sourceURL) + ': ' + t.line + (t.function ? ' (in function ' + t.function +')' : ''))
+    })
+  }
+  console.error(msgStack.join('\n'))
+  phantom.exit(503)
+}
+
+page.onLoadStarted = function() {
+  loading = true
+  clearTimeout(load_timeout)
+}
+
+page.onLoadFinished = function() {
+  loading = false
+}
+
+page.onResourceRequested = function(requestData, networkRequest) {
+  var match = requestData.url.match(/\.css/g)
+  if (requestData.url == "https://dev.botframework.com/bots" ) {
+    on_step = 3
+    networkRequest.abort()
+  }
+}
+
+var steps = [
+  function() {
+    loading = true
+    load_timeout = setTimeout(function(){ loading = false }, 10000)
+    console.log("[INFO] Connecting to https://login.microsoftonline.com...")
+    page.open("https://login.microsoftonline.com/common/oauth2/authorize?response_type=id_token+code&redirect_uri=https%3A%2F%2Fdev.botframework.com%2F&client_id=abfa0a7c-a6b6-4736-8310-5855508787cd&resource=https%3A%2F%2Fmanagement.core.windows.net%2F&scope=user_impersonation+openid&nonce=8e87e9b7-db47-4187-08d4-9ce5cf4cd678&site_id=500879&response_mode=form_post&state=%2Flogin%3FrequestUrl%3D%252Fbots")
+    return 0
+  },
+  function() {
+    loading = true
+    load_timeout = setTimeout(function(){ loading = false }, 20000)
+    retval = page.evaluate(function(username, password) {
+      userid = document.getElementById("cred_userid_inputtext")
+      userpwd = document.getElementById("cred_password_inputtext")
+      submit = document.getElementById("cred_sign_in_button")
+      if ( userid == null || submit == null || userpwd == null ) { return 500 }
+      userid.value = username
+      userpwd.value = password
+      submit.click()
+      setTimeout(function(){document.getElementById("cred_sign_in_button").click()}, 10000)
+      return 0
+    }, username, password)
+    if ( retval == 0 ) {
+      console.log("[INFO] Verifying email address...")
+    } else {
+      console.log("[ERROR] Could not connect to Microsoft!!! Will try again in a second...")
```

Comment:
Yes, I am able to go to the link. I'm wondering if it is my work network. I tried sshing into a linode instance and I'm getting a different error. The Network page is empty.
![screen shot 2017-05-23 at 12 56 31 pm](https://cloud.githubusercontent.com/assets/26485947/26366119/e03c58f0-3fb7-11e7-9844-75e1f3fad378.png)
![screen shot 2017-05-23 at 12 59 21 pm](https://cloud.githubusercontent.com/assets/26485947/26366121/e2128456-3fb7-11e7-9bdb-399ee2608ab3.png)